conf/npm/generateentrypoints.js

```diff
@@ -0,0 +1,72 @@
+#!/usr/bin/env node
+
+const fs = require('fs');
+const { ALL_LANGUAGES } = require('../i18n/constants');
+
+/**
+ * Add entry points to css bundles to the export field
+ *
+ * @param {Object} exportJSON
+ * @param {string} lang
+ */
+function addCSSBundleEntries (exportJSON) {
+  const cssEntries = {
+    './css': './dist/answers.css',
+    './rtl-css': './dist/answers.rtl.css'
+  };
+  Object.assign(exportJSON.exports, cssEntries);
+}
+
+/**
+ * Add entry points to js and template bundles to the export field for default language en
+ *
+ * @param {Object} exportJSON
+ * @param {string} lang
+ */
+function addDefaultLanguageBundleEntries (exportJSON) {
+  const bundleEntries = {
+    '.': './dist/answers-umd.js',
+    './modern': './dist/answers-modern.js',
+    './modern.min': './dist/answers-modern.min.js',
+    './template': './dist/answerstemplates.compiled.min.js'
+  };
+  Object.assign(exportJSON.exports, bundleEntries);
+}
+
+/**
+ * Add entry points to js and template bundles to the export field for the provided language
+ *
+ * @param {Object} exportJSON
+ * @param {string} lang
+ */
+function addLanguageBundleEntries (exportJSON, lang) {
```

Comment:
nit: Could this be a getter instead which just takes in a language and returns an object? We could use the spread operator to add the object to the exports. That would let us avoid having this function need to know about the shape of exportJSON